src/aof.c

```diff
@@ -2244,7 +2244,7 @@ int rewriteAppendOnlyFileRio(rio *aof) {
     int j;
     long key_count = 0;
     long long updated_time = 0;
-    dbIterator *dbit = NULL;
+    daIterator *dait = NULL;
```

Comment:
I don't like introducing acronyms for brevity unless they are common, since it just increases the time it takes for folks to understand something. DB is a very common one and I have no other association outside of Database, DA to me is a District attorney (It's also yes in russian).

Reply:
i'm open to suggestions but i don't think "db" is a good choice because it should be an abstract DS (in addition, it doesn't necessarily represent a database, because also we use it to hold all volatile keys, which are just a subset of the whole database)

Reply:
maybe `dictArr` or `darr`?
i.e. we don't want to invent a new a 2 letter abbreviation, but we're ok to create a new term / class name as long as it's easily distinguishable / grep-able, right?
  
we said we're gonna use `didx` instead of `slot`, right?
and then `count` or `dcount` instead of `slot_conut`?

i'd vote for:
* `dictArr` for function prefix
* `didx` instead of `slot`
* `dcount` for length

Reply:
ok, but i prefer `darr` as function prefix

objections?

Reply:
i'm ok with it (4 letters is better than 2).
let's wait for Madelyn's input.

Reply:
@madolson please comment on the above terminology, and maybe generally what you feel about merging this PR.

Reply:
i was ok with `dictArray`, but i actually like the suggestion of `redisKV`.
how about `kvstore` instead? (not to mention redis in the name)

Reply:
@oranagra @madolson changed to `kvstore`
can i resolve this comment?